developer-model-service/catalog-info.yaml

```diff
@@ -0,0 +1,2036 @@
+# Adapted from: https://raw.githubusercontent.com/benwilcock/backstage-catalogs/main/ai-asset-catalog.yml
+####################################
+# COMPONENTS
+# A Component represents a model server that's deployed, containing metadata about it
+####################################
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: developer-model-service
+  annotations:
+    backstage.io/techdocs-ref: dir:./
+  description: 'vllm and 3scale-based model service providing models for developer tools.'
+  links:
+    - url: https://model-service.apps.domain.com
+      title: Access
+      type: website
+      icon: WebAssett
+    - url: https://ibm-granite-8b-code-instruct-vllm.apps.domain.com
+      title: API URL
+      type: website
+      icon: WebAsset
+  tags:
```

Comment:
Hmm, not sure I follow? What do you mean by both things side to side, and the location of tag?

Reply:
OK, let me try to clarify.  I very well may be missing a fundamental point.


So the schema in https://github.com/redhat-ai-dev/model-catalog/blob/6b67aae6d9cc71f1354b2a3df29940cfc99e99ba/developer-model-service/catalog-info.yaml currently looks different to me than the schema in https://docs.google.com/document/d/1xQy-srzIakk_7fdN5_5AjlbZyXlR035Map8X_VCWYDs/edit#heading=h.lemn1vuieimr based on how I read it.

And you have the statement `But the actual implemented format may vary - as we determine how we want to represent this information on RHDH / Backstage`

Taking that:

 - `Metadata` lines up.

- Now, `Model` as I read it, was a new field in the new, proposed, API.  Or is it now that the`Model` fields there are existing fields in the `Component`, `Resources`, `API` entities that already exist?  I *think* it is the latter, if I interpret the kind vs. type distinction correctly.  But I am not sure.

If it is the latter, it might be helpful in your table in the google doc for the `Model` for example to map those to whatever `Component` / `Resources` / `API` backstage entity fields they are suppose to map to.

Does that help?  Or am I really out of sync, not understanding, something fundamental here?

thanks

Reply:
I think I may have just discovered that https://docs.google.com/document/d/1ArB8w-TawZ_DuMbZoukk4inVsAgqHntB8yN2lttwjyo/edit#heading=h.vb3kc1uzrgdu does the mapping I am talking about ..... not sure which google docs is more up to date, should I look at them in tandem, or should I focus only on https://docs.google.com/document/d/1ArB8w-TawZ_DuMbZoukk4inVsAgqHntB8yN2lttwjyo/edit#heading=h.vb3kc1uzrgdu @johnmcollier ?

Reply:
Yeah I'm pretty sure it is the latter now after (re)reviewing the implementation doc @johnmcollier 

I'l resolve the thread.  Sorry for the confusion.